src/main/render/types.ts

```diff
@@ -27,7 +27,7 @@ interface VideoInput {
   files: string[]
   screenStyle: '....' | '|..' | '_..'
   interval: [number, number]
-  resolution: Resolution[]
+  resolution: Resolution
```

Comment:
The way Vik has implemented VideoInput is that there are multiple files, with multiple resolutions - the file and it's resolution are indexed at the same point in the arrays. Is this a conflict or have you maintained the current code?
@VikaasVarma @CoreyGair